packages/mgt-spfx-utils/README.md

```diff
@@ -0,0 +1,157 @@
+# Utilities for SharePoint Framework Web Parts using Microsoft Graph Toolkit
+
+[![npm](https://img.shields.io/npm/v/@microsoft/mgt-spfx-utils?style=for-the-badge)](https://www.npmjs.com/package/@microsoft/mgt-spfx-utils)
+
+![SPFx 1.15.2](https://img.shields.io/badge/SPFx-1.15.2-green.svg?style=for-the-badge)
```

Comment:
SPFx 1.16.1 is now available. Should we make sure we are in sync with them?

Reply:
Yeah. that's probably a good idea. And for mgt-spf too...

Reply:
Upgraded spfx to 1.16.1
As a side effect I had to force resolution of microsoft-graph-client to 3.0.2 as that's the highest version available as an external in SharePoint